oslabs-beta/DockerLocal
# Env template

## Description:
- These changes prepare the app for launch. A user can enter a user id and github access token into an env file to access their repos.
# Changes I Made:
- Login button on SignIn.tsx checks if user id and access token is entered
- Helper function getUsernameAndToken gets username and token from env file instead of cookies
- Server doesn't use passport.js
- Remove close button from clone repos modal so user doesn't close while repos are cloning
- Change graphQL query to fetch private repos
- Fix typescript errors

## How to Test:
- create a personal access token on github and give all permissions
- enter your github userid and access token into env.ts on the root directory of our app
- npm install and npm start
- all functionality should work
